ZCRMiOS/Classes/ZohoCRMSDK.swift

```
@@ -0,0 +1,135 @@
+//
+//  ZohoCRMSDK.swift
+//  ZCRMiOS
+//
+//  Created by Sarath Kumar Rajendran on 06/09/18.
+//  Copyright © 2017 zohocrm. All rights reserved.
+//
+
+import UIKit
+
+public class ZohoCRMSDK {
+	
+	public static let shared = ZohoCRMSDK()
+	private var isVerticalCRM: Bool = false
+	private var zcrmLoginHandler: ZCRMLoginHandler = ZCRMLoginHandler.init()
+	private var zvcrmLoginHandler: ZVCRMLoginHandler = ZVCRMLoginHandler.init()
+	
+	private init() {
+		
+		if let file = Bundle.main.path(forResource : "AppConfiguration", ofType: "plist" ) {
```

we need to throw an error if we can't read Appconfiguration.plist